launch_ros/launch_ros/parameter.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for a description of a Parameter."""
+
+import collections.abc
+
+from typing import Any
+from typing import Optional
+from typing import Text
+from typing import Tuple
+from typing import TYPE_CHECKING
+
+from launch import LaunchContext
+from launch import SomeSubstitutionsType
+from launch import SomeSubstitutionsType_types_tuple
+from launch import Substitution
+from launch.utilities import ensure_argument_type
+from launch.utilities import normalize_to_list_of_substitutions
+from launch.utilities import perform_substitutions
+
+if TYPE_CHECKING:
+    from .parameters_type import SomeParameterValue
+    from .parameters_type import EvaluatedParameterValue  # noqa: F401, false positive.
+
+
+class Parameter:
+    """Describes a ROS Parameter."""
+
+    def __init__(
+        self,
+        *,
+        name: SomeSubstitutionsType,
+        value: 'SomeParameterValue',
+        value_type: Any = None
```

Comment:
Seems there are very well defined set of types it can be below, so is `Any` the right thing to use here?